packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -398,13 +387,52 @@ export class Telemetry {
     Telemetry.trackCommandEvent(extraProps);
   }
 
-  private static trackCommandEvent(extraProps?: Record<string, any>) {
-    const props: Record<string, any> = {
-      eventName: CommandEventName,
+  private static trackEvent(telemetryItem: coreOneDS.ITelemetryItem) {
+    // Populate Part A
+    telemetryItem.ver = "4.0"; // Current Common Schema version
+    telemetryItem.time = new Date().toISOString();
+    telemetryItem.iKey = RNW_1DS_INSTURMENTATION_KEY;
+
+    const projectPropString = JSON.stringify(Telemetry.projectProp);
+    const versionPropString = JSON.stringify(Telemetry.versionsProp);
+
+    // Populate "common" properties into Part B.
+    telemetryItem.baseData = {
+      common: {
+        device: {
+          id: Telemetry.commonProperties.deviceId,
+          architecture: Telemetry.commonProperties.deviceArchitecture,
+          platform: Telemetry.commonProperties.devicePlatform,
+          locale: Telemetry.commonProperties.deviceLocale,
+          numCPUs: Telemetry.commonProperties.numCPUs,
+          totalMemory: Telemetry.commonProperties.totalMemory,
+          diskFreeSpace: Telemetry.commonProperties.deviceDiskFreeSpace
+        },
+        nodeArchitecture: Telemetry.commonProperties.nodeArchitecture,
+        ciCaptured: Telemetry.commonProperties.ciCaptured,
+        ciType: Telemetry.commonProperties.ciType,
+        isMsftInternal: Telemetry.commonProperties.isMsftInternal,
+        sampleRate: Telemetry.commonProperties.sampleRate,
+        isCliTest: Telemetry.commonProperties.isTest,
+        sessionId: Telemetry.commonProperties.sessionId,
+        commandName: Telemetry.commonProperties.commandName
+      },
+      // Set project and versions props, belonging to Part B.
+      baseType: {},
+      project: projectPropString,
```

Comment:
Is there a reason why you're stringifying these here?

Reply:
Tests were failing due to failed JSON parsing (i.e. not a JSON object), so I converted some props into JSON.

On second thought, we don't really need to make them JSON and then parse to verify those fields. So, I just avoided doing JSON stuff both in product and test code, and tests are passing anyway.